vlayer/ethereum/circuits/lib/src/README.md

```diff
@@ -0,0 +1,49 @@
+# Slots
+
+This is a [Noir](https://noir-lang.org) library for slots.
+
+The library consists of functions:
+
+- ```rust
+  pub(crate) fn mapping(slot: Bytes32, key: Bytes32) -> Bytes32;
+  ```
+
+  Calculates slot number of value corresponding to a mapping `key`. `Slot` argument is the storage location of the mapping calculated after applying storage layout rules.
```

Comment:
Slot number doesn't make sense. Slot itself is a number.
Calculates THE slot
Of A value

Reply:
How slot if calculated is irrelevant in the context of this function. It deals with calculating it's return value. You can mention that it concatenates & hashes

Reply:
I wanted to include the explanation of what is this slot argument, because we need to emphasise the difference between this argument and the one with the same name in the function mentioned later.

Reply:
There will be a link to documentation about solidity storage layout where concatenation and hashing will be explained. So I don't think we need to repeat that here.